microsoft-edge/devtools-guide-chromium/experimental-features.md

```diff
@@ -0,0 +1,69 @@
+---
+description: The latest experimental features in Edge DevTools
+title: Experimental features
+author: hxlnt
+ms.author: raweil
+ms.date: 06/15/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, experiment
+---
+
+# Experimental features
+
+The Edge DevTools periodically offers experimental features that are released while still in development. By enabling an experiment, you are able to test and [provide early feedback](#Providing-feedback-on-experimental-features) on these features before they are released broadly.
+
+Experiments are available in all channels of Edge. Use the Edge Canary channel to get the very latest experiments.
+
+## How to enable experiments
+
+To enable an experiment, open DevTools by pressing `Control+Shift+I` (Windows) or `Command+Option+I` (macOS). Then, open Settings by clicking the gear icon at the top right of DevTools or by pressing `F1`.
+
+On the left side of the Settings panel, there is a tab called "Experiments." Click this tab to see all available experiments. Select the checkboxes next to experiments you would like to try, then close and reopen the DevTools.
+
+![List of experiments in DevTools Settings](./media/experiments-devtools.png)
+
+Note that some experimental features are works in progress and may cause performance issues. You can remove an experiment by unchecking the box next to that experiment and reloading the DevTools as before.
+
+## Current experimental features
+
+The following sections outline some of the experimental features available in Edge now.
+
+### "Enable new CSS grid debugging features"
+
+This feature offers improved on-page visualizations for websites using CSS grid layouts.
+
+![CSS grid debugging feature](./media/experiments-grid.png)
```

Comment:
For this screenshot, could we show the tools opened too or does it not show well given screen real estate? (also wondering if like the screenshot below we could show more content and less of the URL bar).

Reply:
I tried a few different crops, but I think this one gives the overlays the best visibility for now without the image being too blurry or hard to read. (I'd actually like to update this image entirely once some of the new features land, so I'd consider it temporary anyway.)